ecole-pratique-des-hautes-etudes-sciences-historiques-et-philologiques.csl

```diff
@@ -0,0 +1,622 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" page-range-format="expanded" demote-non-dropping-particle="sort-only" default-locale="fr-FR" xmlns="http://purl.org/net/xbiblio/csl">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>École Pratique des Hautes Études - Sciences historiques et philologiques (French)</title>
+    <title-short>EPHE-SHP</title-short>
+    <id>http://www.zotero.org/styles/ecole-pratique-des-hautes-etudes-sciences-historiques-et-philologiques</id>
+    <link rel="self" href="http://www.zotero.org/styles/ecole-pratique-des-hautes-etudes-sciences-historiques-et-philologiques"/>
+    <link href="http://www.zotero.org/styles/invisu" rel="template"/>
+    <link href="http://www.zotero.org/styles/le-tapuscrit-note" rel="template"/>
+    <link href="http://www.zotero.org/styles/traces" rel="template"/>
+    <author>
+      <name>Anne-Sophie Bessero</name>
+    </author>
+    <author>
+      <name>Laurent Garrigues</name>
+    </author>
+    <category citation-format="author"/>
+    <category citation-format="note"/>
```

Comment:
Travis tries to explain that `<style class="in-text" ... >` and `<category citation-format="author"/>` must be compatible with each other.

With `class="note"`, `citation-format` must also be "note".

(see http://docs.citationstyles.org/en/stable/primer.html#citation-formats for the different citation formats, and how CSL groups them as being of the "note" or "in-text" type)